MoonDAO/docs/Projects/Project System.md

```diff
@@ -35,28 +34,30 @@ MoonDAO Projects consist of Project Contributors and a Project Lead. Each Projec
 **Step 1: Post in the “[ideation](https://discord.com/channels/914720248140279868/1027658256706961509)” channel on Discord.** You don’t need to have a fully formed idea for a project yet, and you can be pretty informal at this stage. There are two things to articulate in the ideation channel: 
 1.) Problem to be solved.
 2.) Proposed solution.
+This is meant to help YOU get feedback from people on the idea, offer alternative solutions, and allow for informal coordination around ideas. Anyone can post about ideas regardless of their background in the DAO, even if they aren’t yet a part of our community.
 
-This is meant to help YOU get feedback from people on the idea, offer alternative solutions, and allow for informal coordination around ideas. Anyone can post about ideas regardless of their background in the DAO, even if they aren’t yet part of our community.
+**Step 2: Proposal Submission.** A Project Proposal ([MoonDAO Project Proposal Template](https://docs.google.com/document/d/1p8rV9RlvFk6nAJzWh-tvroyPvasjjrvgKpyX8ibGX3I/edit#heading=h.wq6048m686gd)) must be filled out and submitted [here](https://moondao.com/propose) **by the second Thursday of the Quarter.**
 
-**Step 2: Proposal Submission.** A Project Proposal ([MoonDAO Project Proposal Template](https://docs.google.com/document/d/1p8rV9RlvFk6nAJzWh-tvroyPvasjjrvgKpyX8ibGX3I/edit#heading=h.wq6048m686gd)) must be filled out. The proposal must be submitted **24 hours before** the Senate meeting. If the proposal does not follow its relevant template, then it will not be considered valid and will not go forward for a Senate vote. Any pre-commitments in terms of compensation for team members must be expressed in ETH. See our Proposal Process for more information on how proposals are passed. You can submit your proposal [here](https://moondao.com/propose)
+**Step 3: Senate Review and Approval.** The Senate will convene and review incoming proposals and provide feedback the next day. Proposals may be edited up to **48 hours** before the third Thursday of the Quarter. After the editing deadline is over, the Senate will approve or reject the proposals in accordance with the Constitution. If a proposal passes it will move on to the next step.
 
-## Start
+**Step 4: Townhall Presentation and Member House Vote.** Project contributors MUST be present during the townhall to pitch their project proposal to the community. The member house will then vote on proposals in accordance with the Constitution. The top 50% of proposals that quarter will be funded ranked in order of voting power allocated to them. If less than six proposals are considered, the top three proposals will be funded. If the cumulative budget of accepted proposals surpasses 3/4 of the quarterly budget, then the lowest voted proposal will not be funded until the budget is less than 3/4 the quarterly budget. Project contributors must abstain from voting on their own project.
```

Comment:
I don't see how the vote actually works here?

I think it's a new process so it should be described somewhere.

Reply:
Yes there will be a separate Senate meeting for giving feedback to all incoming proposals. This one will not be recorded, attended only by the Senate and EB leads, and no vote will happen during this meeting. It is intended to be a long-form discussion and feedback session, with the outcome being comments and feedback on each of the proposals by senate members.

This format also allows for Senators and EB leads to discuss proposals freely with candid discussion about the pros/cons without inadvertently offending the authors of the proposals, but then structuring feedback on the proposal itself in a constructive and helpful way.

Again, no vote happens here, it's just intended to be a discussion and written feedback session for proposals.